ingestors/maven/maven_central.go

```diff
@@ -0,0 +1,34 @@
+package maven
+
+import (
+	"github.com/librariesio/depper/data"
+	log "github.com/sirupsen/logrus"
+	"time"
+)
+
+const mavenCentralSchedule = "1 1 * * MON,THU"
```

Comment:
Maybe let's just do these twice a day or something? And extra few runs can't hurt, but minimizing us missing a run.